packages/mdx-embed/src/components/whimsical/whimsical.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { FunctionComponent } from 'react';
+import { GeneralObserver } from '../general-observer';
+
+function getPadding(aspectRatio: string) {
+  const regExp = /^([1-9]\d*):([1-9]\d*)$/;
+  const [, width, height] = regExp.exec(aspectRatio) || ['', '1', '1'];
+
+  return {
+    paddingTop: `${(100 * parseInt(height, 10)) / parseInt(width, 10)}%`,
+  };
+}
```

Comment:
Note: I didn't want to replace `getPadding` from `utils`. In case of Whimsical it makes sense to allow any aspect ratio, since every diagram has its own shape.